# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 2fbec9faf49c
# Full Hash: 2fbec9faf49c7b3389b485a7b836173ba2dbf5cc
# Author: stransky <stransky@redhat.com>
# Date: 2024-12-10 16:36:00
# Regressor Bug: 1934497
# File Overlap Count: 2
# Description:
#   Bug 1934497 [Wayland] Implement wl_surface abstraction as WaylandSurface r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D230847
# ==============================================================================

diff -r f6ea1323c158 -r 2fbec9faf49c modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue Dec 10 07:12:50 2024 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Dec 10 07:12:51 2024 +0000
@@ -17639,6 +17639,12 @@
   type: bool
   value: false
   mirror: always
+
+# Use SW emulation of frame callback, for testing only
+- name: widget.wayland.emulate-frame-callback
+  type: bool
+  value: false
+  mirror: once
 #endif
 
 #ifdef MOZ_WIDGET_GTK